content/teams/bug-byte/index.md

```diff
@@ -0,0 +1,132 @@
+---
+type: "team"                   
+title: bug-byte
+name: "bug-byte"
+github-link: "https://github.com/om-12/srm-bug-byte.git"
+description: "i am new in web development.persuing btech from mnnit"
+Team-Quote: "try try but don't cry"
+Team_Tech_Stack: "full stack"
+Application_Type: "web development"
+No_Of_Member: "3"
+---
+
+Hey ! We are Team bug-byte. We are here to win. and we also love open source.
+
+---
+
+## Project's Overview 
+
+_**Theme-Name**_ : develope for good
```

Comment:
theme is incorrect .. mention the correct theme.   ASAP. as you are already late